libs/counterfactuals/src/lib/CounterfactualChart.tsx

```diff
@@ -361,11 +363,13 @@ export class CounterfactualChart extends React.PureComponent<
                 )}
               </div>
             </div>
-            <LocalImportanceChart
-              rowNumber={this.state.selectedPointsIndexes[0]}
-              currentClass={this.getCurrentLabel()}
-              data={this.props.data}
-            />
+            <div className={classNames.localImportance}>
```

Comment:
why was this put in an extra div?

Reply:
This just a temp fix to solve style issue for new added highchart, will work on modify stylesheets for all the dashboard